reference/configuration/framework.rst

```diff
@@ -515,7 +1169,7 @@ Whether or not to enable the ``translator`` service in the service container.
 fallback
 ........
 
-**default**: ``en``
+**type**: ``string`` **default**: ``'en'``
 
 This option is used when the translation key for the current locale wasn't found.
```

Comment:
It should actually be "isn't" or "is not" :). The wasn't/was not form would be right if we said "The option *would* be used..."

Oh silly English :p